src/data/navigation/sections/graphql.js

```diff
@@ -1317,6 +1317,10 @@ module.exports = [
     title: "Reference (2.4.7)",
     path: "/graphql/reference",
   },
+  {
```

Comment:
Would prefer to have this nested under "Reference (2.4.7)" with "2.4.6" as the heading.